src/target/xtensa/xtensa.h

```diff
@@ -222,11 +254,23 @@ static inline struct xtensa *target_to_xtensa(struct target *target)
 
 int xtensa_init_arch_info(struct target *target,
 	struct xtensa *xtensa,
-	const struct xtensa_config *cfg,
 	const struct xtensa_debug_module_config *dm_cfg);
 int xtensa_target_init(struct command_context *cmd_ctx, struct target *target);
 void xtensa_target_deinit(struct target *target);
 
+static inline void xtensa_stepping_isr_mode_set(struct target *target,
```

Comment:
These are not used. We missed something?  If not, better to delete them.

Reply:
Not sure how these wound up in there.  Not part of NX either.  Removing.